spec/inquisition/issue_spec.rb

```diff
@@ -1,2 +1,53 @@
 RSpec.describe Inquisition::Issue do
+  let(:valid_attributes) do
+    { level: Inquisition::Issue::LEVELS[:low], file: 'file', line: 123, runner: 'runner', message: 'foo bar' }
+  end
+
+  let(:valid_attributes_with_different_runner) do
+    { level: Inquisition::Issue::LEVELS[:low], file: 'file', line: 123, runner: 'runner_2', message: 'foo bar' }
+  end
+
+  let(:valid_attributes_with_different_level) do
+    { level: Inquisition::Issue::LEVELS[:high], file: 'file', line: 123, runner: 'runner', message: 'foo bar' }
+  end
+
+  let(:invalid_attributes) { { level: 'invalid', file: 'file', line: 123, runner: 'runner', message: 'foo bar' } }
+
+  let(:valid_issue) { described_class.new(**valid_attributes) }
+  let(:different_level_valid_issue) { described_class.new(**valid_attributes_with_different_level) }
+  let(:different_runner_valid_issue) { described_class.new(**valid_attributes_with_different_runner) }
+
+  describe '#initialize' do
```

Comment:
Propose to do next,
```ruby
context 'when the unknown level is passed' do
  it do
    expect { described_class.new(level: :unknown }.to raise_error(...)
  end
end
```

instead of 
```ruby
describe '#initialize' do
  ...
end
```
what do you think?